language/move-lang/src/interface_generator.rs

```diff
@@ -60,7 +60,7 @@ pub fn write_to_string(compiled_module_file_input_path: &str) -> Result<(ModuleI
     let mut public_funs = module
         .function_defs()
         .iter()
-        .filter(|fdef| fdef.is_public)
+        .filter(|fdef| matches!(fdef.visibility, Visibility::Public))
```

Comment:
Fully pattern match, it will make migrating this easier as we add visibilities (or break out into a helper for something like `is_externally_visible` )